feature-libs/organization/account-summary/components/details/document/account-summary-document.component.html

```diff
@@ -0,0 +1,269 @@
+<ng-container *ngIf="accountSummary$ | async as accountSummary">
+  <div>
+    <!-- HEADER -->
+    <div class="cx-account-summary-document-header">
+      <h2>
+        {{ 'orgAccountSummary.document.header' | cxTranslate }}
+      </h2>
+    </div>
+
+    <!-- BODY -->
+    <div class="cx-account-summary-document-body">
+      <cx-account-summary-document-filter
+        [documentTypeOptions]="documentTypeOptions"
+        [initialFilters]="_queryParams"
+        (filterListEvent)="filterChange($event)"
+      >
+      </cx-account-summary-document-filter>
+
+      <ng-container
+        *ngIf="
+          accountSummary.pagination &&
+            accountSummary.pagination.totalResults &&
+            accountSummary.pagination.totalResults > 0;
+          else noDocument
+        "
+      >
+        <div class="cx-account-summary-document-sort top">
+          <label class="cx-account-summary-document-form-group form-group">
+            <span>
+              {{ 'orgAccountSummary.sortBy' | cxTranslate }}
+            </span>
+            <cx-sorting
+              [sortOptions]="sortOptions"
+              (sortListEvent)="changeSortCode($event)"
+              [selectedOption]="accountSummary.pagination.sort"
+              placeholder="{{ 'orgAccountSummary.sortBy' | cxTranslate }}"
+              [ariaLabel]="'orgAccountSummary.sortDocuments' | cxTranslate"
+              ariaControls="cx-account-summary-document-table"
+            ></cx-sorting>
+          </label>
+          <div
+            class="cx-account-summary-document-pagination"
+            *ngIf="
+              accountSummary.pagination &&
+              accountSummary.pagination.totalPages &&
+              accountSummary.pagination.totalPages > 1
```

Comment:
Although that is logical (since it's already enclosed in a check for accountSummary.pagination), the build fails complaining that it could potentially be undefined. The shorthand accountSummary?.pagination?.totalPages also doesn't work unfortunately.